node/database-cleaner/src/main.js

```diff
@@ -0,0 +1,19 @@
+import AppwriteService from './appwrite.js';
+
+export default async ({ req, res, log, error }) => {
+  throwIfMissing(process.env, [
+    'APPWRITE_DATABASE_ID',
+    'APPWRITE_API_KEY',
+    'RETENTION_PERIOD_DAYS'
+  ]);
+
+  const appwrite = new AppwriteService();
+
+  const collections = await this.listAllCollections(process.env.APPWRITE_DATABASE_ID);
+
+  for (const collection of collections) {
+    await this.cleanCollection(process.env.APPWRITE_DATABASE_ID, collection.$id);
+  }
+
+  return res.send('Cleaning finished.', 200);
```

Comment:
Can we respond with res.empty() and log this message instead?

Reply:
added